pkg/apis/networkaddonsoperator/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -236,6 +241,22 @@ func (in *NetworkAddonsConfigStatus) DeepCopy() *NetworkAddonsConfigStatus {
 	return out
 }
 
+// DeepCopyInto is an autogenerated deepcopy function, copying the receiver, writing into out. in must be non-nil.
```

Comment:
This have being copy pasted and modified or generated with operator-sdk generate k8s ?

Reply:
generated by the operator sdk

Reply:
ack